oidc-controller/api/db/session.py

```diff
@@ -23,6 +28,32 @@ async def init_db():
     auth_session.create_index([("pres_exch_id", ASCENDING)], unique=True)
     auth_session.create_index([("pyop_auth_code", ASCENDING)], unique=True)
 
+    with open(
+        (
+            Path(__file__).parent.parent / "authSessions" / "sessiontimeout.json"
```

Comment:
This path should be made an environment variable, so a file that is NOT in the application working folder can be passed. This can be the default value if nothing is provided.